.github/workflows/release.yaml

```diff
@@ -0,0 +1,45 @@
+# Prefect Operator release workflow.
+name: Release
+
+on:
+  release:
+    types: [published]
+
+permissions: {}
+
+jobs:
+  build-and-upload-manifests:
+    permissions:
+      contents: write
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: jdx/mise-action@v2
+      - name: Build and upload manifests
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          cd config/manager && kustomize edit set image controller=${{ github.ref_name }}
```

Comment:
We'll need to update this and/or the manifest's `image` field to match the image registry value.

Reply:
Yep, we can do `prefecthq/prefect-operator` (https://hub.docker.com/u/prefecthq)

Reply:
Sounds good, will set this up after the long weekend 👍🏼